src/containerapp/HISTORY.rst

```diff
@@ -2,6 +2,10 @@
 
 Release History
 ===============
+0.3.37
```

Comment:
I think you should use `upcoming`, and not release a new version 0.3.37 in this PR.

Reply:
done